packages/esm-service-queues-app/src/queue-screen/queue-screen.component.tsx

```diff
@@ -1,15 +1,69 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { DataTableSkeleton } from '@carbon/react';
 import { useTranslation } from 'react-i18next';
 import { useActiveTickets } from './useActiveTickets';
 import PatientQueueHeader from '../patient-queue-header/patient-queue-header.component';
 import styles from './queue-screen.scss';
+import { useSelectedQueueLocationUuid } from '../helpers/helpers';
 
 interface QueueScreenProps {}
 
 const QueueScreen: React.FC<QueueScreenProps> = () => {
   const { t } = useTranslation();
-  const { activeTickets, isLoading, error } = useActiveTickets();
+  const { activeTickets, isLoading, error, mutate } = useActiveTickets();
+  const speaker = window.speechSynthesis;
+  const [isSpeaking, setIsSpeaking] = useState(false);
+  const selectedLocation = useSelectedQueueLocationUuid();
+  const locationFilteredTickets = activeTickets;
```

Comment:
Did you mean to filter `activeTickets` by location here? If not, then we probably don't need the alias and we should use `activeTickets` directly.

Reply:
i did this because ill be making a follow up later to filter by location, am waiting for  someone to include the locaion uuid in the response so that i can utilize that,,,

Reply:
Best to not include it until then

Reply:
well gotten